src/librustc_target/spec/windows_games_msvc_base.rs

```diff
@@ -0,0 +1,36 @@
+use crate::spec::{LinkArgs, LinkerFlavor, TargetOptions};
```

Comment:
Similar to `windows_uwp_msvc_base.rs` except that we link against `onecore.lib` instead of `mincore.lib`.

`mincore.lib` is a Windows 8 umbrella library, whereas `onecore.lib` is a Windows 10 library. Since the games partition is new to Windows 10 it made sense to use the newer umbrella library.

See here for more info on umbrella libraries: https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-umbrella-libraries

Reply:
I can add the above as a comment in code if it would be useful.